server/index.js

```diff
@@ -49,10 +49,10 @@ process.on('SIGINT', () => {
 
 process.on('uncaughtException', err => {
   logger.fatal(err.message)
-  process.exit(1)
+  setTimeout(() => process.exit(1), 2000)
```

Comment:
Remind why is there a timeout here?

Reply:
By rethinking about the original problem. we don't need it. It's a sort of a problem in the devl